### PR TITLE
Windows release 0.7.0

### PR DESCRIPTION
**Pull request by achoum**

### CODE REVIEW

File: config/setup.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import setuptools
 from setuptools.dist import Distribution
 
-_VERSION = "0.1.6"
+_VERSION = "0.7.0"
```
Comment (ianspektor) [inline config/setup.py:4]: add modifying the version here in tools/RELEASE.md, step 2?

File: tools/build.bat [hunk 2]
```diff
@@ -12,13 +12,18 @@
 cls
 setlocal
 
-set TEMPORIAN_VERSION=0.1.6
+set TEMPORIAN_VERSION=0.7.0
```
Comment (ianspektor) [inline tools/build.bat:15]: same for here

**Review by ianspektor — APPROVED** [review]

File: config/setup.py (hunk 1 above)
Comment (achoum) [reply config/setup.py:4]: *Done

File: tools/build.bat (hunk 2 above)
Comment (achoum) [reply tools/build.bat:15]: *Done

**Pull request merged**